### ISSUE TITLE
Fix invalid option warning for secondary option in font-family-no-missing-generic-family-keyword

### ISSUE THREAD

**Issue by subzey** [stylelint/stylelint#3038]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug. The rule `font-family-no-missing-generic-family-keyword` should have a way to specify secondary options, but currently it throws.

> Which rule, if any, is this issue related to?

`font-family-no-missing-generic-family-keyword`

> What CSS is needed to reproduce this issue?

Any css, even the empty file.

> What stylelint configuration is needed to reproduce this issue?

```json
{
	"rules": {
		"font-family-no-missing-generic-family-keyword": [true, {
			"severity": "error"
		}]
	}
}
```

> Which version of stylelint are you using?

`8.3.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

The configuration above should have the rule "font-family-no-missing-generic-family-keyword" enabled as an error.

> What actually happened (e.g. what warnings or errors you are getting)?

The following error is shown:

```shell
Invalid Option: Unexpected option value "[object Object]" for rule "font-family-no-missing-generic-family-keyword
```

**Comment by hudochenkov** [comment]
Thank you for reporting and using a template.

We're taking care of this problem in https://github.com/stylelint/stylelint/pull/3039.
